tests/config_fallback_test.py

```
@@ -0,0 +1,63 @@
+    # Save original state
+    original_modules = sys.modules.copy()
+
+    try:
+        # Remove config module if already imported
+        if "commit_check.config" in sys.modules:
+            del sys.modules["commit_check.config"]
+
+        # Make tomllib unavailable by raising ImportError
+        def mock_import(name, globals=None, locals=None, fromlist=(), level=0):
+            if name == "tomllib":
+                raise ImportError("No module named 'tomllib'")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)